src/LanguageServer/Protocol/Handler/Rename/PrepareRenameHandler.cs

```diff
@@ -33,7 +33,7 @@ public LSP.TextDocumentIdentifier GetTextDocumentIdentifier(LSP.PrepareRenamePar
         var position = await document.GetPositionFromLinePositionAsync(linePosition, cancellationToken).ConfigureAwait(false);
 
         var symbolicRenameInfo = await SymbolicRenameInfo.GetRenameInfoAsync(
-            document, position, cancellationToken).ConfigureAwait(false);
+            document, position, allowRenameInGeneratedDocument: false, cancellationToken).ConfigureAwait(false);
```

Comment:
CAn you rename this to includeSourceGenerated.  That's the name we use for other apis that want to conditionally include SG docs or not.